packages/core/src/components/tooltip/tooltip.variables.scss

```diff
@@ -3,8 +3,8 @@
 
 $arrow-width: 6px;
 $opacity: 1;
-$padding-vertical: units.$u2;
-$padding-horizontal: units.$u1;
+$padding-vertical: units.$u1;
+$padding-horizontal: units.$u2;
```

Comment:
😄

Reply:
Ja er was even iemand in de war geloof ik ;)